docs/administration/troubleshooting-published-data-packages.md

```diff
@@ -0,0 +1,92 @@
+---
+parent: Administration Tasks
+title: Troubleshooting published data packages
+nav_order: 21
+---
+# How to troubleshoot published data packages
+
+This documentation explains how users can diagnose issues with all existing published data packages using our new troubleshooting tool. The tool performs a series of checks to verify the integrity and compliance of data packages. The scope of this tool includes both data packages that have been successfully published and those that have failed to publish (packages that initiated the publication process but did not succeed). Specifically, it targets data packages with their Attribute-Value Units (AVUs) including `org_publication_status` of `OK`, `Retry`, `Unrecoverable`, or `Unknown`. Note, the `org_` prefix is defined by the constant variable of `UUORGMETADATAPREFIX` stored in `constants.py` in the ruleset.
+
+Alternatively, the tool can diagnose a specific data package when provided with its name.
+
+**Requirement:** Python 3 or higher required, Yoda version 1.10 or later, rodsadmin user
+
+## Check Steps
+
+The tool performs the following checks:
+
+### Metadata Schema Conformance
+
+This step verifies that the metadata of the data package conforms to the associated schema.
+
+
+### System AVUs Verification
+
+This step checks whether the data package has the expected system Attribute-Value Units (AVUs). It does this by comparing AVUs that start with `org_publication` against the expected AVU keys (ground truth). The check results reveal if there are missing or unexpected AVUs, which will be printed to the terminal and the logfile.
+
+### DOI Registration Status
+
+This step checks the registration status of both `versionDOI` (if available) and `baseDOI` using the DataCite API. It retrieves the DOIs from the package's metadata AVUs and sends API requests to DataCite to verify if these DOIs are registered.
+
+### Landing Page Integrity
+
+This step compares the contents of the local landing page file with the remote landing page to ensure they match. It does this by sending a URL request to download the HTML of the data package's landing page and comparing it with the local HTML file. Note that if there is no internet connection, you should enable the `offline` mode. In offline mode, this step checks if the local landing page file exists but does not verify the correctness of its content.
+
+### Combined JSON Integrity
+
+This step checks the integrity of the combined JSON file by verifying its URL online and confirming the existence of the file. It accomplishes this by checking if the metadata JSON sent to OAI-PMH server can be found in the OAI-PMH repository. In offline mode, it only checks whether package's `-combi.json` file exists locally
+
+
+## Commands
+
+The tool can be used with various options as detailed below.
```

Comment:
We either need to explain here that you should cd to the tools directory first before running these commands, or provide the absolute path to the script in all these examples.